_posts/2018-11-15-using-vault-with-strimzi.md

```diff
@@ -48,7 +48,7 @@ Now, in order to bring your own CA, you need to do this yourself and add the nec
 
 Using Vault to create your own CA arguably gives you more control over your TLS configuration.
 
-Consider a Strimzi Kafka cluster with a Kafka client you have developed. The cluster has been configured with a TLS listener and as such requires that clients authenticate themselves using TLS.
+Consider a Strimzi Kafka cluster with a Kafka client you have developed. The Strimzi cluster has been configured with a TLS listener and as such requires that clients authenticate themselves using TLS.
```

Comment:
Actually I would always say a Kafka cluster and not  Strimzi cluster or a Strimzi deployed Kafka cluster or a Kafka cluster deployed by Strimzi and so on ...
Btw thinking more on this sentence and re-reading the blog post, you are not using TLS client authentication.
The article explains how to use Vault for generate a cluster CA right? It's needed for encryption only (so what is called just TLS server authentication). You are not discussing a way for enabling a TLS mutual authentication (both client and server).